scrapi/consumers/calpoly/README.md

```diff
@@ -0,0 +1,8 @@
+CalPoly
+=======
+
+Consumer for the Digital Commons @ CalPoly API for the SHARE project
```

Comment:
Maybe, though I think readmes with information such as example API requests and other useful bits of information could be helpful for people trying to extend the consumer/start new consumers